Source/Core/Common/CommonFuncs.cpp

```diff
@@ -22,27 +22,37 @@
 
 #ifdef _WIN32
 #include <windows.h>
+#define strerror_r(err, buf, len) strerror_s(buf, len, err)
 #endif
 
-// Generic function to get last error message.
-// Call directly after the command or use the error num.
+constexpr size_t BUFFER_SIZE = 256;
+
+// Generic function to get last strerror message.
 // This function might change the error code.
-std::string GetLastErrorMsg()
+std::string StrErrorMsg()
 {
-  const size_t buff_size = 256;
-  char err_str[buff_size];
+  char error_message[BUFFER_SIZE];
 
-#ifdef _WIN32
-  FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM, nullptr, GetLastError(),
-                 MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), err_str, buff_size, nullptr);
-#else
   // We assume that the XSI-compliant version of strerror_r (returns int) is used
   // rather than the GNU version (returns char*). The returned value is stored to
   // an int variable to get a compile-time check that the return type is not char*.
-  const int result = strerror_r(errno, err_str, buff_size);
+  const int result = strerror_r(errno, error_message, BUFFER_SIZE);
   if (result != 0)
     return "";
-#endif
+  return std::string(error_message);
+}
 
-  return std::string(err_str);
+#ifdef _WIN32
+// Generic function to get last error message.
+// It uses GetLastError on Windows rather than strerror.
```

Comment:
I would suggest removing this line and changing the previous line to something like `// Generic function to get last Win32 error message.`